.azure-pipelines/publish.yml

```diff
@@ -0,0 +1,106 @@
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      # https://aka.ms/MicroBuild
+      name: VSEngSS-MicroBuild2022-1ES
+      os: windows
+    sdl:
+      suppression:
+        suppressionFile: $(Build.SourcesDirectory)\.azure-pipelines\guardian\SDL\.gdnsuppress
+    stages:
+    - stage: BuildPackageSign
+      displayName: Build, Package & Sign
+      jobs:
+      - job: BuildPackage
+        displayName: Build & Package
+        steps:
+        - task: UseDotNet@2
+          displayName: 'Use .NET 8 SDK'
+          inputs:
+            packageType: sdk
+            version: 8.x
+        - task: MicroBuildSigningPlugin@3
+          inputs:
+            signType: real
+            feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+        # We need to download the driver first, so we can build
+        - task: DotNetCoreCLI@2
+          displayName: Download the driver
+          inputs:
+            command: 'run'
+            arguments: '--project $(Build.SourcesDirectory)/src/tools/Playwright.Tooling/Playwright.Tooling.csproj -- download-drivers --basepath $(Build.SourcesDirectory)'
+        - task: DotNetCoreCLI@2
+          displayName: Build Playwright CLI
+          inputs:
+            command: 'build'
+            projects: |
+              src/Playwright.CLI/Playwright.CLI.csproj
+              src/Playwright/Playwright.csproj
+              src/Playwright.NUnit/Playwright.NUnit.csproj
+              src/Playwright.MSTest/Playwright.MSTest.csproj
+              src/Playwright.TestAdapter/Playwright.TestAdapter.csproj
+            arguments: '-c $(BuildConfiguration)'
+        # This (dotnet pack) will internally sign it via the MicroBuildSigningPlugin
+        - task: DotNetCoreCLI@2
+          displayName: Pack Playwright CLI
+          inputs:
+            command: 'pack'
+            packagesToPack: |
+              src/Playwright.CLI/Playwright.CLI.csproj
+              src/Playwright/Playwright.csproj
+              src/Playwright.NUnit/Playwright.NUnit.csproj
+              src/Playwright.MSTest/Playwright.MSTest.csproj
+              src/Playwright.TestAdapter/Playwright.TestAdapter.csproj
+            packDirectory: '$(Build.ArtifactStagingDirectory)/nuget'
+            versioningScheme: 'off'
+        - task: 1ES.PublishBuildArtifacts@1
+          displayName: Copy NuGet Artifacts to Build Results
+          inputs:
+            PathtoPublish: '$(Build.ArtifactStagingDirectory)/nuget'
+            ArtifactName: 'drop'
+        - task: 1ES.PublishNuget@1
+          displayName: Publish Microsoft.Playwright{NUnit,MSTest,TestAdapter}
+          condition: eq('${{parameters.doRelease}}', true)
+          inputs:
+            useDotNetTask: false
+            packagesToPush: |
+              $(Build.ArtifactStagingDirectory)/nuget/Microsoft.Playwright.1.*.nupkg
```

Comment:
Note: The reason for the `1.*` after the package name is that we know the end of the package name in order to not match with e.g. the CLI.

e.g. `Microsoft.Playwright` vs. `Microsoft.Playwright.CLI`.